pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: pana
 description: Evaluate the health and quality of a Dart package
-version: 0.21.17
+version: 0.21.18-dev
 repository: https://github.com/dart-lang/pana
 
 environment:
-  sdk: '>=2.17.0 <3.0.0'
+  sdk: '>=2.18.0 <3.0.0'
 
 dependencies:
-  analyzer: '>=3.1.0 <5.0.0'
+  analyzer: '>=4.7.0 <6.0.0'
```

Comment:
Any reason not to go directly to `^5.0.0`?

Reply:
pub.dev is always behind with some dependency or another (usually a code generator).